### ISSUE TITLE
Blank steps when dragging balloon up and down in "Very close to sweater" column

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#420]
From issue #419. 
When dragging up and down "Very close to sweater" I do not hear any progress up dates. I get black steps in between directional changes and "At top." and "At bottom."

I think this is a side effect of the fact that for "Near Sweater" or "Near Wall" we do not get progress updates.

In the case of Near Sweater, we always hear, "Near Sweater". In the case of "Near Wall" and "Very Close to wall" we have the benefit of changes between "wall, upper wall and lower wall."

@jessegreenberg, can vertical dragging alerts in the "Very close to sweater" column work the same as the "Near sweater" column? So, instead of blank steps, we hear "Very close to sweater."

**Comment by jessegreenberg** [comment]
Thanks for making a new issue @terracoda, looking into it.

**Comment by jessegreenberg** [comment]
Hmm, there is some explicit logic to cause this
```js
        // only announce that we are very close to the sweater when moving slowly to the left
        if ( !movingSlowlyLeft ) {
          alert = null;
        }
```

**Comment by jessegreenberg** [comment]
I think this is probably there because we decided including it was too verbose when moving quickly. We could either remove it, or change it to be  something like
```js
        if ( !movingSlowlyLeft || !movingVertically ) {
          alert = null;
        }
```

**Comment by jessegreenberg** [comment]
Usage of the function  that generates this alert is behind a check for when drag velocity hits zero, so  I think it should be safe to completely remove the check for movingSlowlyLeft. Ill test it out.

**Comment by jessegreenberg** [comment]
Should be fixed above, can you please verify @terracoda?

**Comment by terracoda** [comment]
Will verify, thanks! I like you solution.

**Comment by terracoda** [comment]
@jessegreenberg, verified. Sounds good! Thanks.
